inductiva/coastal/README.md

```diff
@@ -9,29 +9,55 @@ is performed using the [SWASH](https://swash.sourceforge.io/) simulator.
 **Note:**  Some of the following tools require the installation of extra dependencies for the coastal package with
 `pip install --upgrade "inductiva[coastal_extra]"`.
 
-### Example
+### Example 1: Simulating waves on a synthetic bathymetry
 
-Note: You can follow this example in a notebook or run [this script](/demos/py_scripts/coastal_area.py), that contains all the code
-snippets exhibited here. Don't forget to insert your API key
-(`inductiva.api_key = "your_api_key"`) for the script to work! (You can get one by
-filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link))
+You can follow this example in your machine to simulate and visualize the wave propagation in a random bathymetry
+that represents a Coastal Area.
 
-The bathymetry that we use can be loaded from a real scenario or randomly generated
-by us. To avoid having to download any data for now, we will start by run a simulation
+Do not forget to insert your API Key (check the [main page](https://github.com/inductiva/inductiva/tree/main#api-access-tokens) to see how get one).
+
+```python
+import inductiva
+
+inductiva.api_key = "YOUR_API_KEY"
+bathymetry = inductiva.coastal.Bathymetry.from_random_depths(x_range=(0,100),
+          y_range=(0,100),
+          x_num=100,
+          y_num=100,
+          max_depth=1,
+          random_seed=12,
+          initial_roughness=1,
+          roughness_factor=0.5,
+          percentile_above_water=20)
+
+scenario = inductiva.coastal.CoastalArea(bathymetry=bathymetry,
+                                             wave_source_location="W",
+                                             wave_amplitude=0.1,
+                                             wave_period=5.5)
+
+task = scenario.simulate(simulation_time=80, output_time_step=1, fps=5)
+
+output = task.get_output()
+
+output.render(movie_path="movie_path.mp4")
+```
+
+To avoid having to download any data for now, we will start by run a simulation
 on synthetic bathymetry, that is procedurally generated using the API.
 
 We start by generating a random bathymetry.
 ```python
 import inductiva
 
-bathymetry = inductiva.coastal.Bathymetry.from_random_depths(
-        x_range=(0, 100),
-        y_range=(0, 100),
-        x_num=100,
-        y_num=100,
-        max_depth=1,
-        random_seed=12,
-        roughness_factor=0.5)
+bathymetry = inductiva.coastal.Bathymetry.from_random_depths(x_range=(0,100),
+          y_range=(0,100),
+          x_num=100,
+          y_num=100,
+          max_depth=1,
+          random_seed=12,
+          initial_roughness=1,
+          roughness_factor=0.5,
+          percentile_above_water=20)
 ```
 
 This bathymetry is defined in a 100x100 grid, that represent a domain 100m
```

Comment:
With this as well.

Reply:
After the example, you can comment a bit more on the bathymetry object if you want and say the methods it contains, like for example the plot... Then you show the pic.